### PR TITLE
MM - 10286: Make ConnMaxLifetime configurable

### PR DESCRIPTION
**Pull request by jkurian**
#### Summary
* Adds a TextSetting component in Database settings of the admin console

#### Ticket Link
https://mattermost.atlassian.net/browse/MM-10286

#### Checklist
- [x] Ran `make check-style` to check for style errors (required for all pull requests)
- [x] Ran `make test` to ensure unit and component tests passed
- [x] Has server changes (https://github.com/mattermost/mattermost-server/pull/9004)
- [x] Has UI changes
- [x] Includes text changes and localization file ([.../i18n/en.json](https://github.com/mattermost/mattermost-webapp/blob/master/i18n/en.json)) updates

### CODE REVIEW

File: components/admin_console/database_settings.jsx [hunk 1]
```diff
@@ -25,7 +25,7 @@ export default class DatabaseSettings extends AdminSettings {
 
     getConfigFromState(config) {
         // driverName and dataSource are read-only from the UI
-
+        config.SqlSettings.ConnMaxLifetimeMilliseconds = this.parseIntNonNegative(this.state.connMaxLifetimeMilliseconds);
```
Comment (jkurian) [inline components/admin_console/database_settings.jsx]: I could have also used this.parseIntNonZero(this.state.connMaxLifetimeMillseconds, 0) but I didn't think it was logical to pass in 0 as the default value to a function called parseIntNonZero. Let me know what you prefer!

**Comment by jkurian** [comment]
Just rebased this PR

File: components/admin_console/database_settings.jsx [hunk 2]
```diff
@@ -37,6 +37,7 @@ export default class DatabaseSettings extends AdminSettings {
 
     getStateFromConfig(config) {
         return {
+            connMaxLifetimeMilliseconds: config.SqlSettings.ConnMaxLifetimeMilliseconds,
```
Comment (lieut-data) [inline components/admin_console/database_settings.jsx]: really minor nit: maybe order this after query timeout to match the display order?

File: components/admin_console/database_settings.jsx (hunk 1 above)
Comment (lieut-data) [reply components/admin_console/database_settings.jsx]: Good call: I like what you've done :)

File: components/admin_console/database_settings.jsx (hunk 2 above)
Comment (jkurian) [reply components/admin_console/database_settings.jsx]: Good catch! Updated

**Pull request merged**